types/cytoscape/index.d.ts

```diff
@@ -3445,13 +3451,13 @@ declare namespace cytoscape {
              * This property can take on the special value label
              * so the width is automatically based on the node’s label.
              */
-            "width"?: number | "label";
+            "width"?: number | "label" | string;
```

Comment:
In this case, we don't need `"label"` part, since it is covered by more general `string` case and will be ignored by any TSC instance, either compiler itself or any IDE.

Reply:
Yeah I left it there for clarity, but you're right. Removed `"label"` in d7798e0d03.